src/CommunityToolkit.Aspire.Marten/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+[assembly: Experimental("CTASPIREMARTEN001", UrlFormat = "https://aka.ms/dotnet/aspire/diagnostics#{0}")]
```

Comment:
I think we'd be better putting this on the extension methods rather than at the assembly as a whole.

For the link, can you use `https://aka.ms/communitytoolkit/aspire/diagnostics#{0}` as that'll take them to our docs on what the code means.

Also, to we need to document that code in the `docs/diagnostics.md` file.